harness/determined/errors.py

```diff
@@ -72,3 +72,12 @@ class SkipWorkloadException(Exception):
     """
 
     pass
+
+
+class UnsupportedException(Exception):
```

Comment:
`UnsupportedException` is a pretty vague name. I'd use 
```
class InvalidModelException(InvalidExperimentException):
```

Reply:
I didn't see this file updated.

Reply:
I must have missed it. updated.